Cargo.toml

```diff
@@ -63,9 +63,13 @@ batch = ["alloc", "merlin", "rand_core"]
 fast = ["curve25519-dalek/precomputed-tables"]
 digest = ["signature/digest"]
 # This features turns off stricter checking for scalar malleability in signatures
-legacy_compatibility = []
+legacy_compatibility = ["curve25519-dalek/legacy_compatibility"]
 pkcs8 = ["ed25519/pkcs8"]
 pem = ["alloc", "ed25519/pem", "pkcs8"]
 rand_core = ["dep:rand_core"]
 serde = ["dep:serde", "ed25519/serde"]
 zeroize = ["dep:zeroize", "curve25519-dalek/zeroize"]
+
+[patch.crates-io.curve25519-dalek]
+git = "https://github.com/dalek-cryptography/curve25519-dalek.git"
+rev = "f460ae149b0000695205cc78f560d74a2d3918eb"
```

Comment:
FYI, this will be stored in Cargo.lock, so in the future you can do `cargo update -p curve25519-dalek` to update it without having to annotate it manually

Reply:
Does that mean that these lines can be removed entirely?

Reply:
The `rev` line, yes. You can bump it using `cargo update -p curve25519-dalek`